akka-http-circe/src/test/scala/de/heikoseeberger/akkahttpcirce/ExampleApp.scala

```diff
@@ -19,9 +19,10 @@ package de.heikoseeberger.akkahttpcirce
 import akka.actor.ActorSystem
 import akka.http.scaladsl.Http
 import akka.http.scaladsl.server.Directives
-import akka.stream.{ ActorMaterializer, Materializer }
+import akka.stream.{ActorMaterializer, Materializer}
+
```

Comment:
Come on, mate, you have reformatted this code. Please simply revert any changes in the circe module.